spring-boot-project/spring-boot-tools/spring-boot-gradle-plugin/src/main/asciidoc/index.adoc

```diff
@@ -1,34 +1,6 @@
 = Spring Boot Gradle Plugin Reference Guide
 Andy Wilkinson
-:doctype: book
```

Comment:
Is this change necessary? I'd like that we keep the change to what's strictly required

Reply:
Okay, I followed the example of spring-boot-docs.

Reverted by 34a545e0a878d07c7a8a03e5dc02412e385211b9.